nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -561,23 +571,39 @@ def transcribe(
                 )
             trcfg = override_config
             trcfg.timestamps = timestamps
-
-        return super().transcribe(audio=audio, override_config=trcfg)
+        # Check if only one audio is provided with string
+        is_one_audio = isinstance(audio, str) and not (audio.endswith("json") or audio.endswith("jsonl"))
+        # Check if it is provided as a list of strings
+        is_one_audio = is_one_audio or (isinstance(audio, list) and len(audio) == 1)
+        # Check if batch_size is one
+        trcfg.enable_parallel_chunking = is_one_audio or (override_config.batch_size == 1)
```

Comment:
Shouldn;t you check all these only when `enable_chunking=True` ? Else if user doesn;t want chunking then this shouldn;t run. Will be useful for parakeet like models